README.md

```diff
@@ -10,10 +10,12 @@ FontAwesome icon set. Works with polybar.
 
 ## Prerequisites:
 
-bspwm (duh), xprop, python3, FontAwesome etc.
+bspwm (duh), xprop, python3(& pip), FontAwesome etc.
 
 ## Installation:
 
+Install requirements: `pip install --user -r requirements.txt`
+
 Run `bspi_listen &` in your `bspwmrc`.
```

Comment:
Ha, I forgot that our entry point is the `bspi_listen` script. Can we make it optionally accept the `--config` argument so it passes it to `bspi.py`? :thinking:

Reply:
`"$@"` should capture that I think

Reply:
Nice catch! I'd totally forgotten about it aswell :smile: . Yep, should be, I'll fix that!